source/MaterialXGenOgsXml/GlslFragmentGenerator.cpp

```diff
@@ -130,26 +171,13 @@ ShaderPtr GlslFragmentGenerator::generate(const string& fragmentName, ElementPtr
     const bool lighting = graph.hasClassification(ShaderNode::Classification::SHADER | ShaderNode::Classification::SURFACE)
```

Comment:
Nitpick can we have a utility to test for graphs which "require lighting" as it's done in > 1 place.